src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -72,13 +84,19 @@ public String getVersion() {
    */
   protected <T extends ApiData> T call(
```

Comment:
Need to also modify the other invocation of `call` (2 functions below) as well as the `request` method

Reply:
I added to the second call function. but it looks like the request method already added the flags to the query strings

Reply:
We still need to add to the request method so that the default flags get picked up. So it should be 

```java
    if (this.flags != null) {
      urlBuilder.addQueryParameter("flags", this.flags);
    }

   // overrides flags defined by class
   if (vehicleRequest.getFlags() != null) {
      urlBuilder.addQueryParameter("flags", vehicleRequest.getFlags());
   }
```